packages/ember/src/components/card.gts

```diff
@@ -0,0 +1,45 @@
+import Component from '@glimmer/component';
+import { on } from '@ember/modifier';
+import { action } from '@ember/object';
+
+interface CardSignature {
+  Element: HTMLDivElement;
+  Args: {
+    // eslint-disable-next-line no-unused-vars
+    onClick?: (evt: MouseEvent) => unknown;
+  };
+  Blocks: {
+    'card-header': [];
+    'card-body': [];
+  };
+}
+
+export default class Card extends Component<CardSignature> {
+  @action
+  onClick(evt: MouseEvent) {
+    evt?.preventDefault();
+    evt?.stopPropagation();
+
+    this.args.onClick?.(evt);
+  }
+
+  <template>
+    <div
+      class="card p-4 shadow-sm"
+      role="button"
+      {{on "click" this.onClick}}
+      ...attributes
+    >
+      {{#if (has-block "card-header")}}
+        <div class="card-header bg-white">
```

Comment:
Not a problem, just thinking out loud - do you think we'll need to consider different header backgrounds? I don't see any in our designs, but thought it may be worth mentioning

Reply:
This class is modifiable via CSS variables, so alternate themes can still control this. We aren't going to change this now.